portfolio/src/main/webapp/recipe-1.html

```diff
@@ -13,7 +13,7 @@
 
       <header></header>
 
-      <body>
+      <body onload="getComments()">
```

Comment:
Before, we had a button to load the comments, now, they'll automatically display on page load

Reply:
done on all recipe pages